src/api.py

```diff
@@ -193,14 +195,22 @@ def recitation(self):
                     editor = "nvim"
                 elif which("vim") is not None:
                     editor = "vim"
+                elif which("notepad") is not None:
+                    editor = "notepad"
                 else:
                     raise EditorNotFound()
             except EditorNotFound:
                 print("Text editor not found; set the 'EDITOR'" +
-                      "environmental variable and try again")
+                  "environmental variable and try again")
                 exit()
 
-        self.filename = self.cache_path / f"{self.target.reference}"
+        if self.is_windows:
+            windowsfilename = f"{self.target.reference}".replace(":", ";")
+            self.filename = self.cache_path / windowsfilename
```

Comment:
[NIT] Snake case for `windowsfilename`?
```suggestion
            windows_filename = f"{self.target.reference}".replace(":", ";")
            self.filename = self.cache_path / windows_filename
```